2-alcance/README.md

```diff
@@ -1,3 +1,112 @@
+	- Presentaciones de CDs, de conciertos y festivales.
+	- Fallecimientos
+- Ofrecer reportajes y entrevistas.
+	- Reportajes sobre las tendencias sociales respecto a distintos géneros musicales.
+	- Reportajes sobre grupos de música con una larga trayectoria.
+	- Entrevistas a grupos o músicos que recientemente hayan empezado.
+- Ofrecer listas de canciones más escuchadas.
+	- Lo más escuchado en rock.
+	- Lo más escuchado en pop.
+	- Lo más escuchado en reguetón. 
+	- Lo más escuchado en música electrónica.
+- Ofrecer videoclips.
+	- Los últimos videoclips sea cual sea el género musical.
+- Dar a conocer los futuros festivales y conciertos mediante una agenda.
+	- Información sobre grupos y músicos participantes.
+	- Fechas y localizaciones de los eventos.
+	- Enlaces para la compra de entradas.
+
+#### Propósito
+El objetivo prioritario de los contenidos será informar al usuario del panorama musical actual, así como informar acerca de acontecimientos del futuro. Sin embargo, al ofrecer listas de músicas y videoclips, también se busca entretener al usuario.
+#### Tono y lenguaje
+Todo el contenido publicado en la web estará relacionado con la música, pero se tratará el tema sin entrar en aspectos muy técnicos y especializados. El lenguaje será comprensible para todo tipo de lectores, entiendan mucho o poco de música.
+Además, todo el contenido informativo tendrá también como objetivo el entretener, por lo que se emplearán frases cortas y precisas para agilizar así la lectura
+#### Fuentes
+La mayor parte del contenido elaborado será creado por los fundadores de la web para así poder ofrecer un contenido específicamente creado pensando en los usuarios. Las listas de canciones también serán de elaboración propia tras consultar plataformas como Spotify y otras listas de las radios más escuchadas en el Estado Español. Sin embargo, los videoclips se cogerán de plataformas como Youtube, ya que son públicos y no requieren pagar por ningún tipo de licencia.
+### Estructura
+#### Primeras decisiones
+- Canal: el canal principal para informar al usuario será la página web. Sin embargo, también se empleará el correo electrónico para poder informar a aquellos usuarios suscritos a la web.
+- Plataforma: el lanzamiento de la página web será diseñada para ordenadores de mesa y portátiles, pero no se descarta que en un futuro se diseñe la web para ofrecerlo para los móviles.
+- Formato: todos los contenidos ofrecidos en la web serán multimedia. Las noticias de actualidad estarán acompañadas por imágenes, las listas de canciones también tendrán fotografías del músico o grupo, los videoclips una breve descripción, y los eventos que estén en la agenda tendrán su correspondiente imagen del cartel del concierto o festival.
+
+#### Tablas de página
+![Agenda](https://github.com/DeustoPWEB2018/proyectoweb-ocio/blob/master/2-alcance/Agenda.jpg)
+![Entrevista](https://github.com/DeustoPWEB2018/proyectoweb-ocio/blob/master/2-alcance/Entrevista.jpg)
```

Comment:
Aquí empezáis a relacionar unos contenidos con otros. Qué se puede relacionar con qué, es algo que habrá que definir también en los requisitos funcionales (entrevistas con eventos de agenda, noticias entre sí...)